package.json

```diff
@@ -39,6 +39,7 @@
     "prepare": "npm run build && husky install",
     "storybook": "storybook dev -p 6006",
     "build-storybook:v1": "storybook build -c .storybook -o ./reservoir/v1",
```

Comment:
What's the reason for keeping this?

Reply:
Hmm you're right, and it actually won't work. I'll remove.